website/src/components/TokenList.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
```

Comment:
i think this looks great, but I wonder if we'd want to align the design of this component with `UtilityListItem`. maybe they're doing different enough things, but I'd be curious corey's thoughts

Reply:
(this would def be a separate ticket)